h2/src/main/org/h2/command/Parser.java

```diff
@@ -4657,7 +4660,8 @@ private Expression readTerm() {
                 read(DOT);
             }
             if (readIf("REGCLASS")) {
-                FunctionAlias f = findFunctionAlias(database.getMainSchema().getName(), "PG_GET_OID");
+                FunctionAlias f = findFunctionAlias(database.getMainSchema().getName(),
+                        identifiersToUpper ? "PG_GET_OID" : "pg_get_oid");
```

Comment:
`database.sysIdentifier()` should be used here, but I'm not sure about the whole such replacement with this function. It doesn't have any real implementation and it isn't defined when PG server is not in use.

Reply:
fixed